src/main/java/org/assertj/core/api/Assumptions.java

```diff
@@ -84,14 +89,29 @@
  */
 public class Assumptions {
 
-  private static final TypeCache<SimpleKey> CACHE = new TypeCache.WithInlineExpunction<>(Sort.SOFT);
-  private static ByteBuddy byteBuddy = new ByteBuddy().with(TypeValidation.DISABLED);
+  private static ByteBuddy BYTE_BUDDY = new ByteBuddy()
+    .with(TypeValidation.DISABLED)
+    .with(new AuxiliaryType.NamingStrategy.SuffixingRandom("Assertj$Assumptions"));
```

Comment:
why do we need a suffix here ?

Reply:
The strategy takes the original class's name and adds a suffix to distinguish it. The default is ByteBuddy but for debugging purposes, it makes sense to add AssertJ as a name.